cachalot/tests/test_utils.py

```diff
@@ -36,7 +37,9 @@ def force_reopen_connection(self):
     def assert_tables(self, queryset, *tables):
         tables = {table if isinstance(table, str)
                   else table._meta.db_table for table in tables}
-        self.assertSetEqual(_get_tables(queryset.db, queryset.query), tables, str(queryset.query))
+        for setting in (True, False):
```

Comment:
@Fasther: `final_sql_check` instead of `setting`

Reply:
Done :)